src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -82,11 +82,11 @@
                                   :data (utils/->json-string post-data)
                                   :on-done (fn [{:keys [success? xhr]}]
                                              (if success?
-                                               (utils/rlog "SUCCESS in import ! trigger re-render of workspace MC list here?")
+                                               ((:onImportFunc props))
```

Comment:
I will go ahead and add a commit to suit the preference.

Are there any other concerns?  Is the PR otherwise good to merge?